packages/router-plugin/src/core/code-splitter/compilers.ts

```
@@ -2,10 +2,11 @@ import * as t from '@babel/types'
 import babel from '@babel/core'
 import * as template from '@babel/template'
 import {
+  generateFromAst,
+  parseAst,
   deadCodeElimination,
```

ESLint

[error] Member 'deadCodeElimination' of the import declaration should be sorted alphabetically.

(sort-imports)